File: Solutions/Java/Remove_characters_from_alphanumeric_string.java
class Solution {
    String removeCharacters(String S) {
        // code here
         String s = "";
        for (int i=0; i<S.length(); i++) {
            if (Character.isDigit(S.charAt(i))) {
                s += S.charAt(i);
            }
        }
        return s;
    }
}

File: Solutions/Python/Remove_characters_from_alphanumeric_string.py
class Solution:
	def removeCharacters(self, S):
		# code here
		a=[]
        for i in S:
            if (i == '0'or i == '1'or i == '2'or i == '3'or i == '4'or i == '5'or i == '6'or i == '7'or i == '8'or i == "9" ):
                a.append(i)
        return "".join(a)
